gallery/src/pageConfig.ts

```diff
@@ -98,8 +99,9 @@ const pageConfig: PageConfig = {
     NxTruncateEllipsis: NxTruncatePage
   },
   'Guidelines': {
-    'Form Validation Guidelines': FormValidationPage,
     'Contributing to the Gallery': ContributingPage,
+    'Styling Components': StylingComponentsPage,
```

Comment:
Not in alphabetical order

Reply:
> Not in alphabetical order

Fixed the whole group: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/e332fe70e7f2ce18db911c2b1ececdbbe94b77f4